popsycle/synthetic.py

```diff
@@ -208,11 +209,11 @@ def perform_pop_syn(ebf_file, output_root, iso_dir,
          Number of edges for the bins (bins = bin_edges_number - 1)
          Total number of bins is (bin_edges_number - 1)**2
 
-    BH_kick_speed : float 
-        Kick speed for BH (in km/s)
+    BH_kick_speed_mean : float 
+        Mean of the birth kick speed of black holes (in km/s) maxwellian distrubution. Defaults to 50 km/s.
 
-    NS_kick_speed : float 
-        Kick speed of NS (in km/s)
+    NS_kick_speed_mean : float 
+        Mean of the birth kick speed of neutron stars (in km/s) maxwellian distrubution. Defaults to 400 km/s based on distributions found by Hobbs et al 2005 'A statistical study of 233 pulsar proper motions'.
```

Comment:
Suggestion: add the ADS link too (https://ui.adsabs.harvard.edu/abs/2005MNRAS.360..974H/abstract)

Reply:
added